report_thesis/src/sections/background.tex

```diff
@@ -0,0 +1,104 @@
+Nevertheless, these missions were deemed a monumental success and advanced our knowledge of the Martian environment~\cite{marsnasagov_vikings}.
+
+Leveraging the knowledge gained from the Viking missions, NASA launched the \gls{mer} mission in 2003 to investigate whether Mars ever had the conditions to support life as we know it.
+The mission landed two rovers, Spirit and Opportunity, on Mars in January 2004, and they quickly discovered clear evidence that water once flowed on Mars.
+However, since water alone is not enough to support life, the next objective was to search for organic material as well~\cite{marsnasagov_observer, marsnasagov_spirit_opportunity}.
+
+The Curiosity rover landed on Mars in August 2012 inside Gale Crater as part of the \gls{msl} mission with this very purpose.
+Thanks to its sophisticated equipment, Curiosity was able to find evidence of past habitable environments on Mars based on chemical and mineral findings early in its mission~\cite{marsnasagov_msl}.
+
+One of the instruments aboard the rover is the \gls{chemcam} instrument, which is a remote-sensing laser instrument used to gather \gls{libs} data from geological samples on Mars.
+\gls{libs} is a technique that enables rapid analysis by using a laser to ablate and remove surface contaminants to expose the underlying material and generate a plasma plume from the now-exposed sample material~\cite{wiensChemcam2012}.
+This plasma plume emits light that is captured through three distinct spectrometers to collect a series of spectral readings.
+These spectra consist of emission lines that can be associated with the concentration of a specific element, and their intensity reflects the concentration of that element in the sample.
+Consequently, a spectra serves as a complex, multi-dimensional fingerprint of the elemental composition of the examined geological samples~\cite{cleggRecalibrationMarsScience2017}.
+
+\subsection{Baseline \& Replica}
+For analyzing Martian geological samples, the \gls{chemcam} team currently uses the \gls{moc} model~\cite{cleggRecalibrationMarsScience2017}.
+This model integrates \gls{pls} and \gls{ica} to predict the composition of major oxides.
+The \gls{pls} and \gls{ica} phases of the \gls{moc} operate in parallel, and their predictions are blended to form the final predictions.
+Though the \gls{moc} model has proven useful, it suffers from limitations in predictive accuracy and robustness.
+
+An overview of the \gls{moc} model is shown in Figure~\ref{fig:moc_pipeline}.
+We provide a detailed description of the \gls{moc} model in \citet{p9_paper}.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.225\textwidth]{images/moc_pipeline.png}
+	\caption{Overview of the \gls{moc} model.}
+	\label{fig:moc_pipeline}
+\end{figure}
+
+In \citet{p9_paper}, we presented our efforts to replicate the \gls{moc} model.
```

Comment:
![image](https://github.com/chhoumann/thesis-chemcam/assets/29108628/d6deecaa-bc99-4bb1-a414-15fec6d5f267)

The figure doesn't separate the text, so we're basically just repeating ourselves here

Reply:
Resolved